Rdmp.Core/CommandExecution/AtomicCommands/ExecuteCommandAddCatalogueToCohortIdentificationAsPatientIndexTable.cs

```diff
@@ -21,13 +21,17 @@ public class ExecuteCommandAddCatalogueToCohortIdentificationAsPatientIndexTable
         public ExecuteCommandAddCatalogueToCohortIdentificationAsPatientIndexTable(IBasicActivateItems activator, CohortIdentificationConfiguration configuration) : base(activator)
         {
             _configuration = configuration;
+
+            UseTripleDotSuffix = true;
         }
 
         public ExecuteCommandAddCatalogueToCohortIdentificationAsPatientIndexTable(IBasicActivateItems activator,CatalogueCombineable catalogue, CohortIdentificationConfiguration configuration):this(activator,configuration)
         {
             _catalogue = catalogue;
             if(!_catalogue.Catalogue.IsApiCall() && !_catalogue.ContainsAtLeastOneExtractionIdentifier)
                 SetImpossible("Catalogue " + _catalogue.Catalogue + " does not contain any IsExtractionIdentifier columns");
+
+            UseTripleDotSuffix = true;
```

Comment:
You don't want this here.  In this constructor you already have a `_catalogue` chosen so there won't be any popups for the user I don't think